Headset.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Headset
+Comment=Native YouTube Player
+GenericName=Headset
+Exec=Headset %U
+Icon=Headset
```

Comment:
How does your icon look like? Is it https://github.com/headsetapp/headset-electron/blob/master/Icon.png or https://github.com/electron/electron/blob/master/default_app/icon.png? When I install it on my Ubuntu env I get the latter. 
this issue https://github.com/headsetapp/headset-electron/issues/30 is what I'm trying to address.